lib/google_drive/api_client_fetcher.rb

```diff
@@ -22,9 +22,10 @@ def initialize(authorization)
       @drive.authorization = authorization
       # Make the timeout virtually infinite because some of the operations (e.g., uploading a large file)
       # can take very long.
-      @drive.client_options.open_timeout_sec = 100_000_000
-      @drive.client_options.read_timeout_sec = 100_000_000
-      @drive.client_options.send_timeout_sec = 100_000_000
+      t = (2**31 - 1) / 1000
```

Comment:
yup, it's the [max value of Int (32-bit)](https://en.wikipedia.org/wiki/2,147,483,647) represented as milliseconds _(a little under 600 hours)_, then divided to get to seconds, serving as the _maximum allowed timeout value_ that Google's API seems willing accept.